eldritch/mythos.py

```diff
@@ -227,6 +295,80 @@ def create_event(self, state):
     return seq
 
 
+class Mythos12(Headline):
+  def __init__(self):
+    super().__init__("Mythos12", "Square", "Unnamable", {"circle"}, {"square", "diamond"})
+
+  def create_event(self, state):
+    seq = super().create_event(state)
+    seq.events.append(events.ReturnToCup(
+        from_places={"University", "Library", "Administration", "Science"}
+    ))
+    return seq
+
+
+class Mythos13(Rumor):
+  def __init__(self):
+    super().__init__(
+        "Mythos13", "Cave", {"slash", "triangle", "star"}, {"hex"}, "Rivertown"
+    )
+
+  def should_fail(self, state):
+    return state.terror >= 10
+
+  def get_interrupt(self, event, state):
+    if self.failed and isinstance(event, events.Mythos):
+      return self.get_failure_interrupt(event, state)
+    if not self.failed and isinstance(event, events.EncounterPhase):
+      return self.get_pass_interrupt(event, state)
+    return None
+
+  def get_failure_interrupt(self, event, state):  # pylint: disable=unused-argument
```

Comment:
I don't think you want to draw an additional mythos card every turn when this rumor is failed - that's a different rumor.